### DISCUSSION TITLE
Is it possible to implement clickthrough?

### DISCUSSION THREAD

**Discussion by chihiros** [wailsapp/wails#2245] [Q&A]
Example
Electron: https://www.electronjs.org/docs/latest/tutorial/window-customization#create-click-through-windows
Tauri: https://tauri.app/v1/api/js/window/#setignorecursorevents

Does Wails have the same function as `setIgnoreMouseEvents` in `Electron` and `setIgnoreCursorEvents ` in `Tauri`?

**Comment by leaanthony** [comment]
Not at the moment

**Comment by hoangqsgianty** [comment]
Any updates?

**Comment by sybaugay** [comment]
wails3 has the option to set this to true
